tests/io/datasets/test_blo.py

```diff
@@ -30,6 +31,14 @@ def default_blo():
     return ds
 
 
+@pytest.fixture(scope='module')
+def default_blo_raw():
+    import hyperspy.api as hs  # avoid importing top level
```

Comment:
Could we can make `hyperspy` et. al optional? I think these tests have a higher probability to fail, as we don't explicitly keep up with changes in all of those projects. A `try..except` with a `pytest.skip` should do the trick, as the comparison tests are now all independent from the other, right?

Same goes for the other formats.

Reply:
Done!